### ISSUE TITLE
UTM would be awesome

### ISSUE THREAD

**Issue by Sirion-Au** [mendhak/gpslogger#1016]
UTM as an addition would make this App the greatest.

**Comment by mendhak** [comment]
Thanks but I don't know what UTM means, addition to what?

**Comment by Sirion-Au** [comment]
UTM Is Universal Transverse Mercator
Probably the most widely used system in metric countries for land use.
https://en.m.wikipedia.org/wiki/Universal_Transverse_Mercator_coordinate_system
https://www.e-education.psu.edu/geog862/node/1817

**Comment by j-schneid** [comment]
Yes, this is the format that is most easily used by humans because the unit is meters and you can imagine something when you read the numbers. Who knows what a certain fraction of a degree is (in a certain place)?
